core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponent.scala

```diff
@@ -809,7 +809,10 @@ trait EntityComponent {
 
     private def validateEntity(entity: Entity): Unit = {
       if (entity.entityType.equalsIgnoreCase(Attributable.workspaceEntityType)) {
-        throw new RawlsExceptionWithErrorReport(errorReport = ErrorReport(message = s"Entity type ${Attributable.workspaceEntityType} is reserved", statusCode = StatusCodes.BadRequest))
+        throw new RawlsFatalExceptionWithErrorReport(errorReport = ErrorReport(
+          message = s"Entity type ${Attributable.workspaceEntityType} is reserved",
```

Comment:
Is this message ever visible to the user? If yes we could expand it a bit
```suggestion
          message = s"Entity type ${Attributable.workspaceEntityType} is reserved for use by Terra and cannot be overwritten.",
```

Reply:
Will give it another pass without mentioning "Terra", as Firecloud and other front ends may end up seeing this message too.